src/app/forms/custom-sections/adr-section/adr-section-edit/adr-section-edit.component.html

```diff
@@ -446,14 +190,11 @@ <h1 class="govuk-label-wrapper">
                       </tbody>
                     </table>
                   } @else {
-                    <input
-                      appNoEmojis
+                    <govuk-form-group-input
                       id="techRecord_adrDetails_tank_tankDetails_tankStatement_productListUnNo-{{ i + 1 }}"
-                      type="text"
-                      name="techRecord_adrDetails_tank_tankDetails_tankStatement_productListUnNo-{{ i + 1 }}"
+                      [width]="FormNodeWidth.XXS"
```

Comment:
```suggestion
                      [width]="FormNodeWidth.XL"
```

Widths should match.